docker/Dockerfile

```diff
@@ -53,14 +53,15 @@ RUN dnf install -y python3 python3-pip git \
 RUN pip3 install --upgrade pip
 RUN pip3 install --upgrade setuptools
 RUN pip3 --no-cache-dir install SQLAlchemy pyyaml pythreader cx_Oracle j2cli
+RUN python3 -m pip install --no-cache-dir --pre rucio[oracle,mysql,postgresql]
```

Comment:
For consistency, can this not be installed via pip and clubbed with the below install call?

Reply:
pip3 can be used directly, although `--pre` flag is also used in similar situations. As there is no other package being installed using this flag, rucio can not be cluubed with other packages.

Reply:
Why do you want to use the `pre-release` package?